lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/BankAtm.java

```diff
@@ -9,14 +9,14 @@
 /** Represents a bank ATM. */
 public class BankAtm {
   private final Map<UUID, Customer> customerById = new HashMap<>();
-  private final Map<String, CheckingAccount> accountByNumber = new HashMap<>();
+  private final Map<String, BankAccountBase> accountByNumber = new HashMap<>();
```

Comment:
I would've expected `BankAccounts` to be the interface you'd use throughout. Program to interfaces, remember?